requirements-dev.txt

```diff
@@ -0,0 +1,14 @@
+ipython
```

Comment:
why do we need ipython?

Reply:
For the `swift-browser-ui shell` command. It `await`s automatically. Nice to have, but not necessarily required. If it's not listed there, then need to remember to install it separately.